frontend/adapted_strength-app/package.json

```diff
@@ -8,7 +8,6 @@
     "@testing-library/user-event": "^13.5.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-router-dom": "^6.17.0",
```

Comment:
is there a reason we remove react-router-dom here?

Reply:
also we probably want to figure out a way to stop updating to a new version each time we push.

Reply:
Yeah, app would not run with the version I had on desktop. Not sure why - no issues occurred before, but compiler found issues with react-router-dom version